config/routes.rb

```diff
@@ -1,6 +1,20 @@
-Rails.application.routes.draw do
-  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
-
-  # Defines the root path route ("/")
-  # root "articles#index"
+Rails.application.routes.draw do 
+  mount Rswag::Ui::Engine => '/api-docs'
+  mount Rswag::Api::Engine => '/api-docs'
+   devise_for :users,
+    controllers: {
+      registrations: :registrations,
+      sessions: :sessions,
+    }
```

Comment:
- According to your API documentation, you've only one endpoint for user administration. But on here, we can observe how you open all devise endpoints. I think the easier way to fix this is just to remove all the unnecessary endpoints.
<img width="1465" alt="image" src="https://user-images.githubusercontent.com/20563252/186780159-a80ff817-28e7-409b-a781-cc02f8219fb8.png">